common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -369,21 +371,74 @@ def test_cancel_editing_key_value(self):
         self.assertNotEqual(
             original_course_display_name,
             new_course_name,
-            ('original course name:{} can not not be equal to unsaved course name {}'.format(
-                original_course_display_name,
-                new_course_name
-            )
+            (
+                'original course name:{} can not not be equal to unsaved course name {}'.format(
+                    original_course_display_name,
+                    new_course_name
+                )
             )
         )
         self.assertEqual(
             self.advanced_settings.get(self.course_name_key),
             original_course_display_name,
-            ('course name from the page should be same as original_course_display_name:{}'.format(
-                original_course_display_name
+            (
+                'course name from the page should be same as original_course_display_name:{}'.format(
+                    original_course_display_name
+                )
             )
+        )
+
+    def test_editing_key_value(self):
+        """
+        Scenario: Test that advanced settings saves the key value, if save button
+        is clicked from notification bar after the editing
+            Given a staff logs in to studio
+            When this user goes to advanced settings page and enters a new course name
+                Then he clicks 'save' button from the notification bar
+            When this user reloads the page
```

Comment:
The test doesn't actually do an explicit page reload, does it?  Same question for the next test; I think you can just remove this line from the scenario descriptions.